### DISCUSSION TITLE
Problem to start a project, 5 errors.

### DISCUSSION THREAD

**Discussion by josymarss** [skiptools/skip#291] [Q&A]
Does Skip work on Intel chips, or on an Xcode version below 16?

**Comment by marcprux** [comment]
Yes, Skip works on both Intel and Xcode 15+. Can you post the errors you are seeing? Does the command `skip checkup` pass?

**Reply by josymarss** [reply]
My Xcode is on version 14 I believe this is the problem


Initializing Skip application idili-app
[✗] Resolve dependencies (1.71s)
[✗] Build idili-app (2.25s)
[✗] Check Swift Package (0.9s)
[✗] The operation couldn’t be completed. (ArgumentParser.ExitCode error 1.)
[✗] Skip 1.1.20 init failed with 4 errors
Output logged to: /tmp/skip-init-2024-11-18T20:46:18Z.txt
See https://skip.tools/docs/faq and https://community.skip.tools for help
Error: 5 errors

**Reply by marcprux** [reply]
Yes, we do require Xcode 15 or higher, as well as macOS 13 or higher. Are you able to upgrade?

**Reply by josymarss** [reply]
Unfortunately no. Thank you so much for your help.
